src/main/java/sirius/kernel/nls/NLS.java

```diff
@@ -131,6 +131,7 @@ public static String getCurrentLang() {
      * @return the language code of the default language
      */
     @Nonnull
+    @Deprecated
     public static String getDefaultLanguage() {
```

Comment:
shouldn't this remain?

Reply:
Hmm... Depends on what the `defaultLanguage` actually, or rather if it is defined by the NLS or the scope settings. I'll look into this during further development